src/core/manager.c

```diff
@@ -4977,12 +4979,16 @@ void unit_defaults_init(UnitDefaults *defaults, RuntimeScope scope) {
 
                 .oom_policy = OOM_STOP,
                 .oom_score_adjust_set = false,
+
+                .allowed_cpus = (CPUSet) {0}
         };
 }
 
 void unit_defaults_done(UnitDefaults *defaults) {
         assert(defaults);
 
+        cpu_set_reset(&defaults->allowed_cpus);
```

Comment:
I don't think this is needed. This function frees resources, it doesn't need to reset variables.